Ansible/ansible_collections/jfrog/platform/roles/artifactory/tasks/install.yml

```diff
@@ -168,6 +168,18 @@
     - artifactory_licenses | length > 0
   notify: Restart artifactory
 
+- name: Set up Artifactory admin account 
+  ansible.builtin.template:
+    src: bootstrap.creds
+    dest: "{{ artifactory_home }}/var/etc/access/"
+    owner: "{{ artifactory_user }}"
+    group: "{{ artifactory_group }}"
+    mode: 0600
+  when: 
```

Comment:
@bbaassssiiee This will run every time as its these variables are defined , we want these to be set to false by default . which is current case so it won't break existing installs , WDYT ?

Reply:
artifactory_user and artifactory_group are the Linux username and group owning the files on disk. Do not confuse these with the administrator logging into Artifactory, not managed up till now. The product set the default credentials to very well known values, with the ris that you internet-facing Artifactory is hacked in no-time.

This PR allow managing the credentials at install time, by defining real values for:

```yaml
#artifactory_admin_username: admin
#artifactory_admin_password: password
```